couchbase/README.md

```diff
@@ -46,25 +46,27 @@ Metadata associated with SignalFx's Couchbase integration can be found <a target
 
 ### USAGE
 
-Below are screen captures of dashboards created for this plugin by SignalFx, illustrating the metrics emitted by this plugin.
+Screen captures below of dashboards created for this plugin by SignalFx illustrate the metrics emitted by this plugin.
```

Comment:
remove directional language. an alternative that i like using is "the following..."

Reply:
done